describe/test_describe.py

```diff
@@ -20,16 +34,14 @@ def _validate_paths(paths: {}):
             raise FileNotFoundError
```

Comment:
better to have a meaningful message like:
f"The artifact '{path}' was not found within the given directory: '{base_folder}'"... Also I would collect all the missing files and then raise a much more full error, otherwise if you fix this one it will fail again on another.